### PR TITLE
[Impeller] Reorder pipeline construction in content context.

### PR DESCRIPTION
**Pull request by chinmaygarde**
Fixes the regression to the first-frame time on the Mokey.

Also add a trace that should allow for easier identification of problematic pipelines.

A more involved job-queue that allows construction jobs to skip the queue was written in https://github.com/flutter/engine/pull/54355. But the Mokey case is not important enough for the additional complexity. See the context in the linked patch.

Screenshot 2024-08-06 at 1 20 26 PM

[start_up_timeline.json.zip](https://github.com/user-attachments/files/16515389/start_up_timeline.json.zip)

### CODE REVIEW

File: impeller/entity/contents/content_context.cc [hunk 1]
```diff
@@ -334,6 +334,24 @@ ContentContext::ContentContext(
...
+  tiled_texture_pipelines_.CreateDefault(*context_, options, {supports_decal});
+  gaussian_blur_pipelines_.CreateDefault(*context_, options_trianglestrip,
+                                         {supports_decal});
+  border_mask_blur_pipelines_.CreateDefault(*context_, options_trianglestrip);
+  color_matrix_color_filter_pipelines_.CreateDefault(*context_,
+                                                     options_trianglestrip);
+  porter_duff_blend_pipelines_.CreateDefault(*context_, options_trianglestrip,
+                                             {supports_decal});
+  vertices_uber_shader_.CreateDefault(*context_, options, {supports_decal});
+
```
Comment (gaaclarke) [inline impeller/entity/contents/content_context.cc:354]: Can you put these in the brackets above?  That's what that section is for, higher priority pipelines.  Maybe I should have added a comment there last time I sorted these.

**Review by gaaclarke — APPROVED** [review]
LGTM modulo sorting these together with the other high priority pipelines

File: impeller/entity/contents/content_context.cc (hunk 1 above)
Comment (jonahwilliams) [reply impeller/entity/contents/content_context.cc:354]: Not all of these shaders should move up top though. The re-ordering of these is fine, but the glyph atlas one should move of

Comment (chinmaygarde) [reply impeller/entity/contents/content_context.cc:354]: Oh, is that what that was for. I'll move and add the comment.

Comment (chinmaygarde) [reply impeller/entity/contents/content_context.cc:354]: > but the glyph atlas one should move of

Elaborate?

Comment (jonahwilliams) [reply impeller/entity/contents/content_context.cc:354]: oops. "up" I mean. Like we should create glyph atlas after solid color et cetera.

Comment (chinmaygarde) [reply impeller/entity/contents/content_context.cc:354]: Oh, I found a few other cases like the rrect blur stuff in the gallery. So at least some of this necessary IMO.

Comment (jonahwilliams) [reply impeller/entity/contents/content_context.cc:354]: The changes you made are fine, I'm suggesting moving the glyph atlas up even higher.

Comment (chinmaygarde) [reply impeller/entity/contents/content_context.cc:354]: Derp. Ok ok. Got it.

Comment (chinmaygarde) [reply impeller/entity/contents/content_context.cc]: Done.

**Review by jonahwilliams — APPROVED** [review]
LGTM

**Pull request merged**